Servers/UI/OJS.Servers.Ui/Controllers/AccountController.cs

```diff
@@ -83,15 +87,29 @@ public async Task<IActionResult> Login([FromBody]LoginRequestModel model)
             }
             else if (this.webHostEnvironment.IsProduction())
             {
-                return this.Unauthorized(GlobalConstants.ErrorMessages.InactiveLoginSystem);
-            }
+                var signInResult = await this.signInManager
+                    .PasswordSignInAsync(model.UserName, model.Password, model.RememberMe, false);
 
-            var signInResult = await this.signInManager
-                .PasswordSignInAsync(model.UserName, model.Password, model.RememberMe, false);
+                if (signInResult.Succeeded)
+                {
+                    var user = await this.userManager.Users.FirstOrDefaultAsync(u => u.UserName == model.UserName);
+                    var isInRole = await this.userManager.IsInRoleAsync(user!, GlobalConstants.Roles.Administrator);
```

Comment:
Better use First() and do not force `user!`

Reply:
With the new logic, there is check if the user is not null , and the "!" is removed.